composer.json

```diff
@@ -10,11 +10,11 @@
   ],
   "license": "MIT",
   "require": {
-    "php": ">=5.5.9",
+    "php": ">=5.6",
     "broadway/broadway": "^1.0",
     "doctrine/doctrine-bundle": "^1.0",
-    "symfony/console": "^2.3|^3.0",
-    "symfony/http-kernel": "^2.3|^3.0"
+    "symfony/console": "^2.7|^3.3|^4.0",
```

Comment:
having read this: https://symfony.com/blog/helping-prepare-for-symfony-4-bundle-support you only have to add |^4.0. No need to reject <3.3 and <2.7. Composer will figure it out for you.

Reply:
The point here to declare in a explicit way the supported version. 
There're breaking changes in the Symfony 4 release. Do you wanna keep compatibility in no official supported versions?

Reply:
I do want to keep supporting Symfony 3.0-3.2. It's a large number of users.

Reply:
I see the same in the FOSUserBundle: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2639/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780